src/network/networktask.h

```diff
@@ -85,7 +85,11 @@ class NetworkTask : public QObject {
             int timeoutMillis = kNoTimeout,
             int delayMillis = kNoStartDelay) = 0;
     /// See also: invokeAbort()
-    virtual void slotAbort() = 0;
+    ///
+    /// timedOut = false: The abort has been triggered explicitly.
+    /// timedOut = true: The abort has been triggered implicitly
+    /// after a client-side timeout expired.
+    virtual void slotAbort(bool timedOut) = 0;
```

Comment:
This is bikeshedding. I already wanted to mention that I was too lazy to introduce an `enum class` before getting feedback on the proposed solution. I assumed that this is obvious.

Reply:
> We already have a code path to deliver timeouts to the calling code.

You still confuse client-side with server-side timeouts. A client-side timeout is an abort, but with an implicit instead of an explicit trigger. Technically, it is not an error, because the task did what you told it to do (according to the timeout parameter provided when starting it). A server-side timeout is an unexpected outcome and an error.

Aborting a request might subsequently let your business code fail. But that's a different level of abstraction.

Reply:
> This is bikeshedding.

Seriously @uklotzde ? Please treat me with respect!

Reply:
The client side timeout is an network error in the same way as the server side timeout. There is no need to pass one over a different signal than the other. In contrast the slotAbort is a direct responds to an explicit immediately abort request. Using that for a Network was wrong and causes the bug I have fixed.  

But anyway this is an implementation detail f the bug fix. 

You have not explained why you experience a bug with it.